app/controllers/admin/organizations_controller.rb

```diff
@@ -106,7 +106,7 @@ def sortable_columns
   end
 
   def kind_for_organizations
-    # Legacy enum issue so excited for TODO: Rails 5 update
+    # TODO: Refactor to incorporate Rails 5 enum changes
     Organization::KIND_ENUM[params[:search_kind].to_sym] || 0
```

Comment:
Question: 

What's the issue alluded to here?

Reply:
I am pretty sure you can just use the string, without having to cast it to the actual integer.

So this line can become slightly simpler and a lot more readable:

```ruby
Organization.pos_kinds.include? ? params[:search_kind] : "no_pos"
```

And the line above this can become `return %w[no_pos does_not_need_pos]`

These updates don't need to be addressed in this PR. I can deal with them afterward. It's mainly stuff that will make things more readable (at least from my perspective)